sdk/identity/identity/src/tokenCache/TokenCachePersistence.ts

```diff
@@ -31,11 +32,20 @@ export class TokenCachePersistence {
       retryDelay: 50
     };
 
+    let extensions: any;
+    try {
```

Comment:
Are we sure that this code path is not hit when user does not opt into the persistent token cache feature? And is this feature on or off by default?

Reply:
this feature is off by default, and this path is not hit unless the user opts in! Here's the code: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/src/msal/nodeFlows/nodeCommon.ts#L58